src/Pages/Main/Components/Nav/Nav.scss

```
@@ -129,45 +129,89 @@ a {
     position: relative;
   }
 
-  .searchContainer {
+  .formGroup {
+    padding-top: 5px;
     input {
       margin: 0px 10px;
       outline: 0px;
       border: none;
       border-bottom: $lightGrey solid 1px;
     }
+  }
 
-    &:hover {
-      .searchTrend {
-        display: block;
-      }
+  .searchButtonIcon {
+    i {
+      padding-left: 10px;
+    }
+  }
+
+  .searchAndRecentContainer {
+    position: absolute;
+    margin-top: 37px;
+    right: 0px;
+    display: flex;
+    border-radius: 15px;
+  }
```

```suggestion
  .searchAndRecentContainer {
    position: absolute;
    right: 0px;
    margin-top: 37px;
    display: flex;
    border-radius: 15px;
  }
```
인접한 속성 끼리는 연달아 써주는 것이 좋습니다.

---

알겠습니다 다른곳에서도 수정했습니다!